libsolidity/ast/AST.h

```diff
@@ -669,7 +669,7 @@ class UsingForDirective: public ASTNode
 		bool _global
 	):
 		ASTNode(_id, _location),
-		m_functions(_functions),
+		m_functionsOrLibrary(std::move(_functions)),
```

Comment:
Oh, I just noticed that this is still called `_functions`...

Reply:
Yeah, I'll reapprove if you change that as well.

Reply:
Fixed.